packages/cli/src/zosfiles/list/List.options.ts

```diff
@@ -28,7 +28,8 @@ export const ListOptions: { [key: string]: ICommandOptionDefinition } = {
         name: "volume-serial",
         aliases: ["vs"],
         description: strings.VOLUME,
-        type: "string"
+        type: "string",
+        stringLengthRange: [1, 6]  // tslint:disable-line:no-magic-numbers
```

Comment:
Wondering if we should revisit linting rules at some point.
:yum:

Reply:
Maybe when we migrate to ESLint we can revisit them 🙂